src/resources/CrawlingModule/CrawlingModuleInterfaces.ts

```diff
@@ -92,19 +92,19 @@ export interface ComponentVersion {
     /**
      * @deprecated Should not be used.
      */
-    databaseVersion: string;
+    databaseVersion?: string;
     /**
      * The version of the Crawling Module.
      */
     maestroVersion: string;
     /**
      * @deprecated Should not be used.
      */
-    securityWorkerVersion: string;
+    securityWorkerVersion?: string;
     /**
      * @deprecated Should not be used.
      */
-    workerVersion: string;
+    workerVersion?: string;
 }
```

Comment:
Making these parameters optional is slightly breaking because `ComponentVersion` is the response type of `CrawlingModule.getMaestroVersions`. I don't think that's too bad though.

Reply:
Important note: The Crawling Module with Docker (pre-1.0, the one that uses these additional versions) has gone beyond deprecated and has officially has had its planned end of life last July: https://coveo.slack.com/archives/C0GQ1HE6B/p1657113320798079

Reply:
@btaillon can we resolve this conversation?